.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: ruby
 rvm:
-  - 2.3.1
+  - 2.4
+  - 2.3
   - 2.2
   - 2.1
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
   matrix:
-    - JEKYLL_VERSION=3.0
+    - JEKYLL_VERSION=3.5
```

Comment:
This is a meaningless change. We are testing against the latest 3.x version, so updating this with each point release is unnecessary.

Reply:
I see.